packages/infra/lib/ihe-stack/ihe-db-construct.ts

```diff
@@ -85,6 +88,12 @@ export default class IHEDBConstruct extends Construct {
       });
     }
 
+    new r53.CnameRecord(this, `${id}PrivateRecord`, {
+      recordName: `db.${config.subdomain}.${domain}`,
+      zone: privateZone,
+      domainName: dbCluster.clusterEndpoint.hostname,
```

Comment:
OOC what is the purpose of this?

Reply:
So we have an easier way of accessing the DB, no need to look for the load balancer name. There's an easier life with known/expected names. 😄